launchable/utils/session.py

```diff
@@ -0,0 +1,70 @@
+import click
+import os
+import json
+from pathlib import Path
+from typing import Optional
+
+session_file_path = Path('~/.config/launchable/session.json').expanduser()
+
+
+class SessionError(Exception):
+    pass
+
+
+def read_session(build_name: str) -> Optional[str]:
+    if not session_file_path.exists():
+        raise SessionError(
+            "No session file. Please write session {}".format(session_file_path))
```

Comment:
When I think about the situation in which the execution gets here, my mental picture is that somebody invoked `launchable subset --build` without ever calling `launchable record build`. I don't think asking users to "please write session $FILENAME" gets them on to the path of solving this problem.

To me, a better error message is "no session information recorded in $FILENAME. Did "launchable record  tests" run?"

Reply:
How does this behavior compared to a situation when a file does exist but no session is recorded for the specified build name?